### PR TITLE
[Windows] Add ImageMagick tool

### PR DESCRIPTION
**Pull request by erik-bershel**
# Description
Add ImageMagick tool to both Windows images. 

#### Related issue:
#6207

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1 [hunk 1]
```diff
@@ -307,3 +307,9 @@ function Get-SwigVersion {
     $swigVersion = $Matches.Version
     return "Swig $swigVersion"
 }
+
+function Get-ImageMagickVersion {
+    ($(magick -version) | Select-String -Pattern "Version") -match "(?<version>\d+\.\d+\.\d+.\d+)" | Out-Null
```
Comment (al-cheb) [inline images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:312]: nitpick: remove $() around magick

Comment (erik-bershel) [reply images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:312]: I'll wait for the build and make changes depending on the result. I may need more than this one.
I have a question about this line. Isn't this version specification redundant or insufficient?
An example of outputting the version from the application for understanding: "Version: ImageMagick 7.1.0-48 Q16 x64 c243c92:20220911"

Comment (miketimofeev) [inline images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1:312]: Maybe a bit easier to read but up to you
`"(?<version>(\d+\.){2}\d+-\d+)"`

**Review by ddobranic — APPROVED** [review]

File: images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1 (hunk 1 above)
Comment (erik-bershel) [reply images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1]: Partially accepted. The dot was replaced with the expected character.

**Pull request merged**